M2/Macaulay2/m2/varieties.m2

```diff
@@ -142,6 +142,8 @@ module CoherentSheaf := Module => F -> F.module
 module SheafOfRings  := Module => F -> module F.ring
 Ideal * CoherentSheaf := (I,F) -> sheaf(F.variety, I * module F)
 CoherentSheaf ++ CoherentSheaf := CoherentSheaf => (F,G) -> sheaf(F.variety, F.module ++ G.module)
+CoherentSheaf.directSum = args -> sheaf((first args).variety,directSum apply(args, F -> F.module))
+directSum CoherentSheaf := directSum @@ sequence
```

Comment:
I'm beginning to really dislike `@@`. Notice how this one method looks significantly less readable than all the others:
```m2
i1 : code methods directSum

o1 = -- code for method: directSum(ChainComplex)
     ../linuxbrew/.linuxbrew/share/Macaulay2/Core/chaincomplexes.m2:579:28-579:
     45: --source code:
     directSum ChainComplex := C -> directSum(1 : C)
     ---------------------------------------------------------------------------
     -- code for method: directSum(CoherentSheaf)
     ../linuxbrew/.linuxbrew/share/Macaulay2/Core/classes.m2:90:47-90:54:
     --source code:
     Function @@ Function := Function => (f,g) -> x -> f g x
     | symbol  class                 value      location of symbol
     | ------  -----                 -----      ------------------                                                 
     | f       MethodFunctionSingle  directSum  ../linuxbrew/.linuxbrew/share/Macaulay2/Core/classes.m2:90:37-90:38
     | g       CompiledFunction      sequence   ../linuxbrew/.linuxbrew/share/Macaulay2/Core/classes.m2:90:39-90:40
     | -- function f:
     | function directSum: source code not available
     | -- function g:
     | function sequence: source code not available
     ---------------------------------------------------------------------------
     -- code for method: directSum(GradedModule)
     ../linuxbrew/.linuxbrew/share/Macaulay2/Core/gradedmodules.m2:222:44-222:61
     : --source code:
     directSum GradedModule := GradedModule => M -> directSum(1 : M)
     ---------------------------------------------------------------------------
     -- code for method: directSum(List)
     ../linuxbrew/.linuxbrew/share/Macaulay2/Core/matrix.m2:324:23-324:47:
     --source code:
     directSum List := args -> directSum toSequence args
     ---------------------------------------------------------------------------
     -- code for method: directSum(Matrix)
     ../linuxbrew/.linuxbrew/share/Macaulay2/Core/matrix.m2:268:22-268:47:
     --source code:
     directSum Matrix := f -> Matrix.directSum (1 : f)
     ---------------------------------------------------------------------------
     -- code for method: directSum(Module)
     ../linuxbrew/.linuxbrew/share/Macaulay2/Core/matrix.m2:290:22-290:47:
     --source code:
     directSum Module := M -> Module.directSum (1 : M)
     ---------------------------------------------------------------------------
     -- code for method: directSum(Option)
     ../linuxbrew/.linuxbrew/share/Macaulay2/Core/matrix.m2:336:22-336:39:
     --source code:
     directSum Option := o -> directSum(1 : o)
```
Just a consequence of #2569 ...